### ISSUE TITLE
How to include layout into an extension shortcut?

### ISSUE THREAD

**Issue by khomutovspace** [datenstrom/yellow#465]
My goal now is to show posts inside of other post content (text). I supposed, i would wrie an extension for this task, which could get one var (for example file name) as an argument and then show a layout, which has already caught posts' title, description and picture by making a page collection using the file name arg. But it always show me PHP error in every way i had tried to complete it. Can anyone help me?

**Comment by schulle4u** [comment]
Have a look at my [Include extension](https://github.com/schulle4u/yellow-extensions-schulle4u/tree/master/include). It can include page content from other pages into the current page, either as teaser or full content. Does this comply your task? 

Steffen

**Comment by khomutovspace** [comment]
Yeah, mostly it is really near to what I need. But I would like you use match and a name of the file, not the location, and get access to it's settings such as title, description and so on, not all the content.

**Comment by schulle4u** [comment]
Then have a look at the [preview extension](https://github.com/datenstrom/yellow-extensions/tree/master/features/preview), maybe you can built upon it. @BSNoSi has created a [similar extension](https://github.com/bsnosi/yellow-extension-lister) you can use for page collections. 

Steffen
